integration_tests/e2e/changeEstablishment.cy.ts

```diff
@@ -15,7 +15,7 @@ context('Change establishment', () => {
   })
 
   // Check current establishment, change establishment and check again
-  it('Should change establishment and redirect to home page', () => {
+  it.skip('Should change establishment and redirect to home page', () => {
```

Comment:
Just skipping this as this whole page will be going in next PR (and it breaks because the links it uses are no longer on the new header)